Makefile.cache

```diff
@@ -69,7 +69,7 @@
 SONIC_COMMON_FILES_LIST :=  $(if $(wildcard cache.skip.common),, .platform slave.mk rules/functions Makefile.cache)
 SONIC_COMMON_FLAGS_LIST :=  $(CONFIGURED_PLATFORM) \
                             $(SONIC_DPKG_CACHE_SOURCE) $(SONIC_DEBUGGING_ON) \
-                            $(SONIC_PROFILING_ON) $(SONIC_ENABLE_SYNCD_RPC)
+                            $(SONIC_PROFILING_ON) $(SONIC_ENABLE_SYNCD_RPC) $(INSTALL_KUBERNETES)
```

Comment:
why this common flag? it does not affect most build packages.

Reply:
it is unclear why you did this, can you explain more on the motivation?

Reply:
It is to generate a new hash for the packages with INSTALL_KUBERNETES enabled. The option only impacts the installer images, right? And there are no cache for the images which are built every time.

Reply:
the common flags list impact every build artifacts.

Reply:
I think the PR is not necessary, the INSTALL_KUBERNETES is only applied to the installer images (SONIC_INSTALLERS), and we never use cache for the images.